sexp.c

```diff
@@ -2190,6 +2190,8 @@ sexp sexp_write_one (sexp ctx, sexp obj, sexp out, sexp_sint_t bound) {
 #endif
     case SEXP_PROCEDURE:
       sexp_write_string(ctx, "#<procedure ", out);
+      if (sexp_procedure_variable_transformer_p(obj))
+        sexp_write_string(ctx, "(variable-transformer) ", out);
```

Comment:
Perhaps omit this, or change it to just `#<variable-transformer ...>`? I mainly used it for debugging `make-variable-transformer`.